plugins/outputs/cloudwatch/cloudwatch_test.go

```diff
@@ -323,45 +323,43 @@ func (svc *mockCloudWatchClient) PutMetricData(input *cloudwatch.PutMetricDataIn
 	return args.Get(0).(*cloudwatch.PutMetricDataOutput), args.Error(1)
 }
 
-func newCloudWatchClient(svc cloudwatchiface.CloudWatchAPI) *CloudWatch {
+func newCloudWatchClient(svc cloudwatchiface.CloudWatchAPI, forceFlushInterval time.Duration) *CloudWatch {
 	cloudwatch := &CloudWatch{
 		svc:                svc,
-		ForceFlushInterval: internal.Duration{Duration: 1 * time.Second},
+		ForceFlushInterval: internal.Duration{Duration: forceFlushInterval},
```

Comment:
I understand the input variables; however, all of the test case is using `time.Second`; therefore, I would say hard code constant would be a good idea unless the test are flakiness.

Reply:
Almost all test functions use the `time.Second` has 2nd parameter.
But the new `TestPublish()` function uses a different value (60 seconds).
That is why I added the 2nd parameter.